packages/lib-classifier/src/components/Classifier/components/QuickTalk/QuickTalk.js

```diff
@@ -52,75 +57,111 @@ function QuickTalk ({
-          <UnorderedList as='ul' flex={false}>
-            {comments.map(comment => {
-              const author = authors[comment.user_id]
-              const roles = authorRoles[comment.user_id]
+        <Box
+          direction='row'
+          flex={false}
+          justify='between'
+          pad='small'
+        >
+          <Anchor
+            a11yTitle={t('QuickTalk.aria.goToTalk')}
+            label={t('QuickTalk.headerLink')}
+            href={subject.talkURL}
+            target='_blank'
+            icon={<Chat />}
+          />
+          <Button
+            a11yTitle={t('QuickTalk.aria.closeButton')}
+            autoFocus={true}
+            icon={<Close size='small' />}
+            onClick={() => setExpand(false)}
+            plain
+          />
+        </Box>
+        <Box overflow='auto' tabIndex='0'>
+          <Box flex={false} pad={{ bottom: 'small', left: 'small', right: 'small' }}>
+            {comments.length > 0 && (
+              <UnorderedList as='ul' flex={false}>
+                {comments.map(comment => {
+                  const author = authors[comment.user_id]
+                  const roles = authorRoles[comment.user_id]
 
-              return (
-                <Comment
-                  key={`quicktalk-comment-${comment.id}`}
-                  comment={comment}
-                  author={author}
-                  roles={roles}
-                />
-              )
-            })}
-          </UnorderedList>
+                  return (
+                    <Comment
+                      key={`quicktalk-comment-${comment.id}`}
+                      comment={comment}
+                      author={author}
+                      roles={roles}
+                    />
+                  )
+                })}
+              </UnorderedList>
+            )}
+            {!comments.length && (
+              <Box
+                background={{ dark: 'dark-1', light: 'light-1' }}
+                margin={{ horizontal: 'none', 'vertical': 'xsmall' }}
+                pad='xsmall'
+              >
+                <Paragraph textAlign='center'>
+                  {t('QuickTalk.subjectHasNoComments')}
+                </Paragraph>
+              </Box>
+            )}
+          </Box>
+        </Box>
+        <Box
+          flex={false}
+          pad={{ horizontal: 'small', vertical: 'xsmall' }}
+        >
           {userId && (
             <PostForm
               postComment={postComment}
               postCommentStatus={postCommentStatus}
               postCommentStatusMessage={postCommentStatusMessage}
             />
           )}
+          {!userId && (
+            <Box
+              background={{ dark: 'dark-1', light: 'light-1' }}
+              margin={{ horizontal: 'none', 'vertical': 'xsmall' }}
+              pad='xsmall'
+            >
+              <Paragraph textAlign='center'>
+                {t('QuickTalk.loginToPost')}
+              </Paragraph>
+            </Box>
```

Comment:
```suggestion
            <Paragraph
              background={{ dark: 'dark-1', light: 'light-1' }}
              margin={{ horizontal: 'none', 'vertical': 'xsmall' }}
              pad='xsmall'
              textAlign='center'
            >
              {t('QuickTalk.loginToPost')}
            </Paragraph>
```

Try this out before committing it, but I think you can use those styling properties directly on the paragraph.

Reply:
I remember running into this problem earlier - I couldn't get the Paragraph to accept certain style props.

With the original `<Box styles='...'><Paragraph ... /></Box>`:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/13952701/176953341-12362dfa-5226-4bf4-bcfe-fa93dfa7a816.png">

With the suggested `<Paragraph styles='...' />`:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/13952701/176953475-e209dd4c-3a45-4d8c-bb4e-4999747aa594.png">

Anyway, check this out, I replaced my login bits with:
```
<Box pad='large' background='red'>
  This is a Box with pad='large' background='red'
</Box>
<Paragraph pad='large' background='red'>
  This is a Paragraph with the same props
</Paragraph>
```

And I got this:
<img width="300" alt="image" src="https://user-images.githubusercontent.com/13952701/176954094-774600e3-0bcb-4e34-8daf-b3749cd62e1b.png">

I thought a Paragraph was supposed to inherit all the basic props of a Box?? Or was this an earlier version of Grommet that I was using? (On Classrooms?) In any case, thank you for the suggestion.

Reply:
I thought Paragraph accepted the same styling props as Box, but obviously not.